power/Power.tcl

```diff
@@ -69,6 +69,19 @@ proc_redirect report_power {
   }
 }
 
+define_cmd_args "report_internal_power_components" { [> filename] [>> filename] }
```

Comment:
why does this require a reporting function? none of the other power types have unique reporting functions.